src/ordsets_ext.erl

```diff
@@ -0,0 +1,34 @@
+% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015-2016 Christopher Meiklejohn.  All Rights Reserved.
+%% Copyright (c) 2007-2012 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ordsets_ext).
+-author("Vitor Enes Duarte <vitorenesduarte@gmail.com>").
+
+-export([equal/2]).
+
+-spec equal(ordsets:ordset(), ordsets:ordset()) ->
+    boolean().
+%% @doc Two sets s1 and s2 are equal if:
+%%      - s1 is subset of s2
+%%      - s2 is subset of s1
```

Comment:
I know about quickcheck but never used, let's see what I can do.
But anyway, maybe even in terms of computation, it's better to iterate one of the sets.